.github/workflows/CI.yaml

```diff
@@ -13,9 +13,9 @@ jobs:
           node-version: 22
       - uses: oven-sh/setup-bun@v1
       - run: make setup
-      - run: make build-lib-js
-      - run: make build-lib-types
-      - run: make build-web
+      - run: make clean build-lib-js
+      - run: make clean build-lib-types
+      - run: make clean build-web
```

Comment:
Are these changes related to typescript vs js? Or are they just riding along?

Reply:
This is the "exercise them in CI" part of the title. If we didn't clean then we wouldn't actually be exercising the optimizations from this PR (that is, we wouldn't catch regressions).